oaknational/Oak-Web-Application
# Feat/copyright on the on the lesson overview page  LESQ-352

## Description

- Adds copyright icon
- Adds licence to lesson overview page

## Acceptance Criteria:

- [ ]  In the blue box with the equipment etc, depending on whether we are showing new or legacy content, display the relevant copyright message (same copy as seen on downloads page)
    - LEGACY For old content the statement should read:
    
    > This content is made available by Oak and its partners and licensed under Oak’s terms and conditions, except where otherwise stated.
    > 
    - NEW For new content the statement should read:
    
    > This content is © Oak National Academy (2023), licensed on [[Open Government Licence version 3.0](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)  except where otherwise stated. See Oak’s terms and conditions.
    >

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1956--oak-web-application.netlify.thenational.academy
Go to legacy and new lesson overview page to see licence message

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
